http-filter-example/header_processor.cc

```diff
@@ -0,0 +1,58 @@
+#include "header_processor.h"
+
+namespace Envoy {
+namespace Http {
+
+    SetHeaderProcessor::SetHeaderProcessor() {}
+
+    int SetHeaderProcessor::parseOperation(std::vector<absl::string_view>& operation_expression) {
+        int err = 0;
```

Comment:
Do we need a var for this or could we just return 0 at the end?

Reply:
Addressed comment in [this PR](https://github.com/DataDog/envoy-filter-example/pull/4).